config/Common.xcconfig

```diff
@@ -0,0 +1,3 @@
+GCC_WARN_UNUSED_PARAMETER = YES
+WARNING_CFLAGS = -Wno-nullability-completeness
+IPHONEOS_DEPLOYMENT_TARGET = 14.0
```

Comment:
Nice!

Reply:
Should we `#include` this in the Jetpack `xcconfig` as well?

Reply:
I was thinking the same thing. From the configuration filename, "WordPress.xcconfig" and "Jetpack.xcconfig", I thought they were for the two targets, in which case they should have similar content. But that's not the case.

As you can see from the configuration settings, "WordPress.xcconfig" is used in the project level, whereas the "Jetpack.xcconfig" is used in the target level. That means, "WordPress.xcconfig" and "Jetpack.xcconfig" both take effects when compiling the Jetpack target.

<img width="665" alt="Screenshot 2023-04-05 at 8 10 42 PM" src="https://user-images.githubusercontent.com/1101828/230021542-e777774d-a5cd-479c-8515-d12bc57a232c.png">

Reply:
> That means, "WordPress.xcconfig" and "Jetpack.xcconfig" both take effects when compiling the Jetpack target.

Gotcha. I can see why that's the case because Jetpack was added afterwards. 🤔 I guess we can live with it for the time being.